<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixDlg</class>
 <widget class="QDialog" name="MatrixDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatrixTreating</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>55</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>状态集:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_state_set">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入符号集:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input_set">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_init_state">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>117</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>唯一初态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>147</y>
     <width>54</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>终态集:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_final_state_set">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>190</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>转换矩阵:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MatrixDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MatrixDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
